core/src/main/java/io/aiven/kafka/tieredstorage/RemoteStorageManagerConfig.java

```diff
@@ -156,6 +172,28 @@ public class RemoteStorageManagerConfig extends AbstractConfig {
             ConfigDef.Importance.HIGH,
             ENCRYPTION_PRIVATE_KEY_FILE_DOC
         );
+
+        CONFIG
+            .define(METRICS_SAMPLE_WINDOW_MS_CONFIG,
+                ConfigDef.Type.LONG,
+                30000,
+                atLeast(0),
```

Comment:
I guess it doesn't make sense to be 0
```suggestion
                atLeast(1),
```